evm/vm/forks/spurious_dragon/__init__.py

```diff
@@ -0,0 +1,101 @@
+from evm import constants
+from evm.exceptions import (
+    OutOfGas,
+)
+from evm.utils.hexadecimal import (
+    encode_hex,
+)
+from evm.utils.keccak import (
+    keccak,
+)
+
+from ..frontier import _execute_frontier_transaction
+from ..homestead import HomesteadVM
+
+from .blocks import SpuriousDragonBlock
+from .opcodes import SPURIOUS_DRAGON_OPCODES
+from .utils import collect_touched_accounts
+
+
+def _execute_spurious_dragon_transaction(vm, transaction):
+    computation = _execute_frontier_transaction(vm, transaction)
+
+    #
+    # EIP161 state clearing
+    #
+    touched_accounts = collect_touched_accounts(computation)
+
+    with vm.state_db() as state_db:
+        for account in touched_accounts:
+            if state_db.account_exists(account) and state_db.account_is_empty(account):
+                vm.logger.debug(
+                    "CLEARING EMPTY ACCOUNT: %s",
+                    encode_hex(account),
+                )
+                state_db.delete_account(account)
+
+    return computation
+
+
+def _apply_spurious_dragon_create_message(vm, message):
+    snapshot = vm.snapshot()
+
+    # EIP161 nonce incrementation
+    with vm.state_db() as state_db:
+        state_db.increment_nonce(message.storage_address)
+
+    computation = vm.apply_message(message)
+
+    if computation.error:
+        vm.revert(snapshot)
+        return computation
```

Comment:
I think you're visually missing an indentation level.  Agreed that this code block is difficult to read.  Just not sure what to do about it right now (I took the liberty of removing the `if vm.logger` conditionals since all of our VM's have a logger.

Reply:
Indeed, I thought the other `return computation` was not inside the `else` block